### PR TITLE
docs(data-950): add notebook for document plugin and resources

### PR DESCRIPTION
**Pull request by hguerni**

### CODE REVIEW

File: recipes/plugins_example_document.ipynb [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d8490f2",
+   "metadata": {},
+   "source": [
+    "This project has two jobs with several tasks of transcription associated. We will be interested in the *Payment information* job, and more precisely the *IBAN* and the *Currency* sub-jobs. With our plugin, we want to be sure that the labelers write the correct values for the two transcriptions (so we are interested only in the `on_submit` handler of the plugin), since we know for sure that the IBAN should start with the 2 letters *FR*, and that the currency should be one of : *EUR*, *USD*. At then end, we also calculate an accuracy and we insert it in the `json_metadata` of the asset.\n",
```
Comment (gheorghetutunaru) [inline recipes/plugins_example_document.ipynb:169]: In the actual plugin it's EURO and DOLLAR, not EUR and USD

File: recipes/plugins_example_document.ipynb [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+      "Requirement already satisfied: python-dateutil>=2.8.1 in /Users/havast/.local/lib/python3.10/site-packages (from pandas->kili) (2.8.2)\n",
+      "Requirement already satisfied: pytz>=2020.1 in /Users/havast/opt/anaconda3/envs/python3/lib/python3.10/site-packages (from pandas->kili) (2022.7)\n",
+      "Requirement already satisfied: numpy>=1.21.0 in /Users/havast/opt/anaconda3/envs/python3/lib/python3.10/site-packages (from pandas->kili) (1.24.1)\n",
+      "Requirement already satisfied: idna<4,>=2.5 in /Users/havast/opt/anaconda3/envs/python3/lib/python3.10/site-packages (from requests->kili) (3.4)\n",
+      "Requirement already satisfied: charset-normalizer<3,>=2 in /Users/havast/opt/anaconda3/envs/python3/lib/python3.10/site-packages (from requests->kili) (2.1.1)\n",
+      "Requirement already satisfied: certifi>=2017.4.17 in /Users/havast/opt/anaconda3/envs/python3/lib/python3.10/site-packages (from requests->kili) (2022.12.7)\n",
+      "Requirement already satisfied: six>=1.5 in /Users/havast/.local/lib/python3.10/site-packages (from python-dateutil>=2.8.1->pandas->kili) (1.16.0)\n"
+     ]
```
[the remark this answers is not in the document]
Comment (hguerni) [reply recipes/plugins_example_document.ipynb:56]: I thought they autodeleted, will do

File: recipes/plugins_example_document.ipynb [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+   "execution_count": 5,
+   "id": "cc56bc57",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the project\n",
+    "\n",
+    "title = \"Demo QA Bot PDF\"\n",
+    "description = \"Demo QA Bot PDF\"\n",
+    "input_type = \"IMAGE\"\n",
```
[the remark this answers is not in the document]
Comment (hguerni) [reply recipes/plugins_example_document.ipynb:103]: Indeed, I will bring it up ont ticket

File: recipes/plugins_example_document.ipynb (hunk 1 above)
Comment (hguerni) [reply recipes/plugins_example_document.ipynb:169]: same here, thanks for the thorough review gheorghe !

File: recipes/plugins_example_document.ipynb [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+    "from kili.client import Kili\n",
+    "import os\n",
+    "import time\n",
+    "import json\n",
+    "import glob\n",
+    "\n",
+    "main_api_key = \"put-your-api-key-here\"\n",
+    "api_endpoint = \"http://cloud.kili-technology.com/api/label/v2/graphql\"\n",
+    "\n",
+    "kili = Kili(api_key=main_api_key, api_endpoint=api_endpoint)"
```
[the remark this answers is not in the document]
Comment (hguerni) [reply recipes/plugins_example_document.ipynb:49]: I did run `pre-commit run --all-files` and it didn't throw me any error, I will rebase

**Review by gheorghetutunaru — APPROVED** [review]

File: recipes/plugins_example_document.ipynb (hunk 4 above)
[the remark this answers is not in the document]
Comment (gheorghetutunaru) [reply recipes/plugins_example_document.ipynb]: My bad, apparently it checks if it's present in `tests/test_notebooks.py` only for notebooks that already have a tutorial in the SDK doc, so here it is skipped

**Review by pmKili — APPROVED** [review]

**Pull request merged**